### PR TITLE
Add remove functionality

### PR DESCRIPTION
**Pull request by arabiu033**
- Modified the CSS file, added some designs
- Customized the JS file for code readability and performance
- Add a new JS file providing the  page functionality
- Use class methods
- Fix linters

### CODE REVIEW

File: src/task.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      }
+
+      ele.textContent = textArea.value;
+      menu.classList.remove('hide');
+      bin.classList.add('hide');
+      textArea.classList.add('hide');
+      ele.classList.remove('hide');
+      li.classList.remove('edit');
+    };
+  }
```
Comment (phelian23) [inline src/task.js:63]: - [ ] Kindly make sure that when a task is edited, the editing persist and the storage is updated

File: src/task.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  remove(ele) {
+    this.tasks = this.tasks.filter((e) => e.index !== +ele.parentElement.id);
+    for (let i = 0; i < this.tasks.length; i += 1) {
+      this.tasks[i].index = i + 1;
+    }
+    localStorage.setItem('listOfTasks', JSON.stringify(this.tasks));
+    ul.removeChild(ele.parentElement);
+  }
```
Comment (phelian23) [inline src/task.js:35]: - [ ] Kindly make sure a task is deleted when the delete icon is clicked

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @arabiu033,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good descriptive PR 👍
✅ Good commit messages 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/task.js (hunk 1 above)
Comment (arabiu033) [reply src/task.js:63]: Yeah, Forget to update the local storage. Thank you sir

File: src/task.js (hunk 2 above)
Comment (arabiu033) [reply src/task.js:35]: Actually based on the design example given to us, you can only delete the task when it's empty.
But I will implement what you said. Thank you.

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @arabiu033  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**